imgcreate/kickstart.py

```diff
@@ -413,7 +413,7 @@ def apply(self, ksnet):
                 raise errors.KickstartError("No --device specified with "
                                             "network kickstart command")
 
-            if (network.onboot and network.bootProto.lower() != "dhcp" and 
+            if (network.onboot and network.bootProto.lower() != "dhcp" and
```

Comment:
Please drop this change from the commit. I know it's for the better, but it would clutter git history for no real reason. If you want to fix this, do some wider cleanup change.